packages/create-nx-workspace/src/utils/nx/ab-testing.ts

```diff
@@ -116,3 +116,17 @@ export async function recordStat(opts: {
     }
   }
 }
+
+function shouldRecordStats(): boolean {
+  const pm = detectPackageManager();
+  try {
+    const stdout = execSync(`${pm} config get registry`, { encoding: 'utf-8' });
```

Comment:
This is not the right command for `yarn 2+` the variable there is... `npmRegistryServer` https://yarnpkg.com/configuration/yarnrc#npmRegistryServer.

Maybe we should put this into the `pmCommands`?